css/jstree.css

```diff
@@ -400,9 +400,23 @@
vux9nzua32P7gCvFOtWrOOZoGJnjEL3IEClGj3LnLba1Q++R2G+cphNFzJg/lKHsyqPDR9cwyNZafRVFYcEIDY1wG2NFRCnvi8O0IRggiFBBGKEETE+7aKbwUoS4hxB0uA5it5SMl8DxFbfoyRH0sx4iMpfvHufTh5MhFNZcVt9j9YF0Lf6VrAeQtXBgQYGRnZ5jm6DpDokCTYfS+EXuRx6DUwX8lHxfkUVJxPRfnFVJjV+TBfPQKz6ohPjnmBFSPBWAmiPKqELOF5tzwxxi1LjHHLEmLc8oQYtyzhebcswfs4p+hQYAxOTAG6m01oKitAU1kBco/GIWrHz/H4jgk4XZQcSH5+gjUGyBhjWm0lbqW1/LKybmDjxiOImSOHf4iGVoIQhAgrQdwOE9wOA9zNRu/WbvBuHQa4nAJcgjNoK0Fuvfj4TviX4oktQH+6c+g1sNeocfL8QZy9UnCb/Bx6DZrKijt0KVzMHDk2bjyCjRuPYNWadYG/Y+bIkbI/tWckQIII9s0Q2l/4v6zdGwF0lZsR9ISbIeQUHbrtVNg//he3O44SIEEQdDssSoAEQZAAKQESBEECpARIEAQJkBIgQRAkQEqABEGQACkBEgRBAqQESBAECZASIEEQJEBKgARBUAKkBEgQBCVAEiBBEJQASYAEQVACJAESBEEJkARIECRASoAkQIIgAVICJAESBAmQEuB/hABvrb8xdu+dOmMjY33E7ByCIAFSAuwyH2D+iBErdJs2Vef99Kfzbu2ML4YNm2/askVzYsCAZ4MpQCljfVvTkX0h7RcSvmHDhk3tMWXKFBkd8AQlQJEF+H3q3j/dF7Xz6srLE7dPXCTaG/RVdt++f6xdvRrIyEDDhx8688LD5f7XCocMkduTk52oqIA9Pb3pmE+CQZFfv5BwxlgkYyxS2i8kvCMl6JPcbWW32cHzPDZs2LCJDnqCEmAnCLDf8kHjFNffMpQ4jiOX/wAPbo6YJ5YA0xhbrg4NBT9gAMy//jVc27bB/P77zvzw8LkF4eFzbR984ERuLpCdDZw9C0dq6s0Tgwc/GwwBMsYijUYTjEYTpFLpE4yxyI6SYHsCdAkCjEYTVCo1AJAECUqAwRDgvase/NWLOc9khb0cPuK25LdiWPROzYra3LodSNRuRop2K94vWYH+K4c8JcYHuJ+xTy4yhhuMQc8YzAMHQli4ELbt2+F4911g507ggw+AvXuB1FTgyBGce+ihLcE4BZVKpU8YjSYojyo7XILtCdADwNpohVqlhrXRCpcgkAQJSoBiCrDPq0Onrj0725JheROrPp+rYfKBQ/xv4kdLh43deun1uviqFYjVLMCWqy/jPdUreC7tsXK2IHyMCB+gJI2x4cmMZVxlDDxjqGcMFsbgfOwxeNasgWftWmD9eiA2Fu5PPsFX48ZlpDE2PJgCfOoPT3a4BO90CuxyCrA2WqFSq6FSeZFKpU+QAAhKgP+mAKULB0fFHB3dtFb1S6y4OAWbv3kGrxbM0rAXfjL0ntceeHRl0W/rXrswHvJTo/DSF+OxuCga0+IjNUw2aLyIY4CSNMaGpzCWrmLMU8sYjIzByhiaH34YrsWLgddfR8vatZ4vR49OT2NsOGNMEmwB3irBYI8J+gTouS0FegBBcMLSaIXRaIK10QqaFCEoAYqRAOcPnRqVOMg0+8QgPHv8XswsCMVzRQMhOzij7OWjT9TOKrwbk3MYJmcxTMpkGLGr93UmHzJB7EkQxhhLZmzoIan0arVEAp4xmHwSdIwYAbz6KtSzZl1OZmxosGaB2xNgawkGOwVOnTpV3p4A0epJlyDA5RRIgAQlQNHGAOcP/dXPdoXWRGczRGV5ZTfjUC9MywnBxAyGiWkME9IY7tsepmEvDBsr6htsLcBeveZ9ddddTRWMoYox+JOghTE0P/IIjEuWNLa+RKb7CXCaDG6PC/+sPCABEpQARZ0Flg/77f07pXWPpjCMS2YYq2QYyzGMSWIYncQw5L1e3zDZ8ImiG9pXSsbkJ3r3btEwhlLGUO6bFKllDAbG0MAYHBMmoH7hwpaDQ4bIO0qAHXkKPH3GdLnH7RH8ic+P94+2wZAESFACFPsyGPmQpwa9K619eB/DqD0+FAz3vh1WymRDxwflDTLGkkNC5h6WSIQzjOE8Y7jKGK4xhmu9e+O6RAK9b3LEEBaGm+PHg587Vzg8ePCc7jYJMmPGDLnb7XbCDbhdLrgFwYvLBZfggksQAhmQBEhQAgzGdYDzhz7747d71Y78WIoRH0lxz/qwa2zufVFBMzRjTBkW9te8gQPxRUgIihnDRcZwPiwMB/r2PZjC2KErjKHalwjr+/cHHx2N/GHDNgVTgB0tv+joaMn06dPlzpaWFpcgQBCccDq9uJwCXE6nd/xPEOChBEhQAgzihdCyob96ZOtI06h3RpbdKfmJKEBJGmPDEsPC/nF44EAUhYbiZEgIUqTS9DTGHkxj7EElY+kXfOOCX991F3LvuecfaYwN624XQk+fPl1ua7I57TY77HYftttxCQIlQIISYBBXgkiYbMh/tZ7tDWYCZIxJ9jN2f1JYWELmgAFQMnbAd6mLf6XI8CTGDpxgDJkSScJ+xu4PymUwnbwU7rHHHpNZrBZXY1Mj2nCzEdbGRlibrLA2WmG32SkBEpQAu9nNECS+tPdiGmMRjDHJLa9F+F570P9ad7sZwtSpU2VOZ4u7ubkFDocDdpsNDrsDzY5mCC1OCIITLsFJkyAEJcBuejssSSv+6Wvd7csslUqfWLVmnXv12vXu1WvXu9a8+RfXmrXrXavXrnevWrPes3rNOs+qNeuwas06WglCUAIMlgC7+k511y+C/xS8b//e4dJ+IeF9+vce3Kd/2OA+d/ce3Kd/78HSfiHhAbzjlVRUlABJgKBvCxUVJUASIBUVFSXALi1AKioqqh6bAKmoqKgoAVJRUVFRAqSioqLqIgmQIAiiO/D/AwDuKUn4ex3n/gAAAABJRU5ErkJggg==);
```

Comment:
Awesome! Could we commit these changes separately in a separate `less` file? Then include the hosted css and js . That way we could more easily update in the future if needed.